crates/context_aware_config/migrations/2024-05-06-133756_custom_types/up.sql

```diff
@@ -0,0 +1,42 @@
+-- Your SQL goes here
+CREATE TABLE jsonschema_types (
+    type_name TEXT PRIMARY KEY,
+    type_schema JSON NOT NULL,
+    created_by TEXT NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    last_modified TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL
+);
+CREATE INDEX IF NOT EXISTS json_schema_types_index ON public.jsonschema_types(type_name);
+CREATE INDEX IF NOT EXISTS json_schema_types_created_at_index ON public.jsonschema_types(created_at);
+CREATE INDEX IF NOT EXISTS json_schema_types_last_modifed_index ON public.jsonschema_types(last_modified);
+INSERT INTO jsonschema_types(type_name, type_schema, created_by, created_at)
+VALUES (
+        'Number',
+        '{"type": "number"}',
+        'user@superposition.io',
+        NOW()
+    ),
+    (
+        'Decimal',
+        '{"type": "number"}',
+        'user@superposition.io',
+        NOW()
+    ),
+    (
+        'Boolean',
+        '{"type": "boolean"}',
+        'user@superposition.io',
+        NOW()
+    ),
+    (
+        'Enum',
+        '{"type": "string", "enum": "$replacement"}',
```

Comment:
Should we put an example array of options here, which can help user understand how to exactly use enum type?

Reply:
Any suggestions?

Reply:
I've added [android, ios]